en/technical-reference/ip-address.md

```diff
@@ -15,19 +15,16 @@ Sandbox Environment:
 103.58.103.177
 ```
 
-In case you are unable to **receive notification from Midtrans**, please ensure that your infrastructure already allow/whitelist the IP above.
+In case you are unable to **receive notification from Midtrans**, please ensure that your infrastructure has whitelisted the above-mentioned IP addresses.
 
 ### API Domain Endpoint
 
-Midtrans API endpoint is distributed and protected with multiple layers of security, so it will not have any specific IP address. Please whitelist our API domain name instead:  
+Midtrans API endpoint is distributed and protected with multiple layers of security, so it does not have any specific IP address. So, please whitelist our API domain names mentioned below.  
 ```
 api.midtrans.com
 app.midtrans.com
 ```
 
-In any case you are unable to **send request to Midtrans**, please ensure that you already whitelisted the domain above.
+If you are unable to **send request to Midtrans**, please ensure that you have whitelisted the above-mentioned domain name.
 
-If it really necessary to whitelist IP address instead of Domain, you can refer to the [following IP address list](https://www.cloudflare.com/ips-v4). Although not recommended.
-
-
-[API IP address range](https://www.cloudflare.com/ips-v4 ':include :type=iframe width=100% height=300px')
```

Comment:
Please do not remove this part, as it will also embedd the IP list on the current page (as iframe)